Makefile

```diff
@@ -139,6 +139,8 @@ define protoc
       --grpc-gateway_out=logtostderr=true:${GOPATH}/src \
       --swagger_out=logtostderr=true,fqn_for_swagger_name=true:. \
       $(1)
+     perl -i -pe 's|argoproj/argo/|argoproj/argo/v2/|g' `echo "$(1)" | sed 's/proto/pb.go/g'`
```

Comment:
This might not be necessary after this PR is merged, but currently it's a chicken-and-egg problem